build.gradle

```diff
@@ -212,7 +212,7 @@ android {
     medicmobilegamma_training {
       dimension = 'brand'
       applicationId = 'org.medicmobile.webapp.mobile.medicmobilegamma_training'
-      buildConfigField "boolean", "IS_TRAINING_APP", 'true'
+      buildConfigField "boolean", "IS_TRAINING_APP", 'false'
```

Comment:
This is `true` in `v1.0.x`.

Reply:
So can we keep it `true` in this commit ?